openitp-good-packaging-for-LAFS_sources-of-nondeterminism.rst

```diff
@@ -186,6 +186,8 @@ This approach can work only if, for each platform, the ➾ operation
 performed by the distributor results in a bytewise-identical binary
 to the ➾ operation performed by the security auditor.
 
+.. This is not strictly true, for example, a "fuzzy fingerprinting" algorithm could produce an identifier using only a subset of the build output bytes, for example by excluding known timestamps fields from a hash calculation.  That seems like a messy route and the one presented here is more sensical, so this is somewhat pedantic.
+
```

Comment:
I think that if you know the subset that should be omitted from the fuzzy fingerprint, you also know how the build should be modified to make it deterministic. So the only advantage would be if you're using tools that cannot be changed to make them deterministic. Actually that might be the case on Windows, so I think I will add a paragraph about it.